docs/business-finder-links-based-facets.md

```diff
@@ -0,0 +1,40 @@
+# EU Exit Business Readiness finder links-based facets
+
+This document explains how the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") uses [content-tagger](https://github.com/alphagov/content-tagger) 'links' to filter the finder and retrieve content from different publishing apps.
+
+## How is this different from the user perspective?
+
+In most finders, search results get more granular as you select values in different facets (the number of results goes down). The business finder search results get more _broad_ as you select values in different facets (number of results goes up). In addition, search results only show content from one publishing app.
+
+The business finder can be sorted by topic ("grouped display"). In this context, a 'topic' is the label of the facet (such as 'Personal data'), except for the 'Sector / Business area' facet, where the _facet value_ (such as 'Aerospace') is the topic. Search results can show content from multiple publishing apps.
```

Comment:
This last sentence ("Search results can show content from multiple publishing apps.") belongs with the paragraph above.

Reply:
Oops.